### ISSUE TITLE
Reported issue using HXCPP debugging with IntelliJ plugin

### ISSUE THREAD

**Issue by jgranick** [HaxeFoundation/intellij-haxe#884]
https://github.com/openfl/openfl/issues/2012

Is there any more information we can help provide? I am sorry there are not more details

**Comment by EricBishton** [comment]
@hereticByte - Arnim, Can you please provide some more information about what you are seeing?  Is this a new issue that did not exist with earlier versions of lime?  What is the exact error?

**Comment by hereticByte** [comment]
@EricBishton,
Hello Eric,

you can find the post here:
[https://community.openfl.org/t/trouble-after-updating-to-openfl-8-4-1/10978](url)

I am not so sure, if there are so many more information to get out of that post.

Regrettably I only can tell, that calling the function _startDebugger()_ leads to an immediate crash without error report. It looks like a _null-pointer error_. 

I have also updated _lime_ to _7.0.0_

As I do not get any insights, and as I am not so familiar with the **hxcpp-debugger**, I can't do further experiments, to limit the critical areas within the framework or within my own project.

But I suggest, that I deliver a _sandbox project_ that crashes on my mashine.

Here is the link to the github repository:

[https://github.com/hereticByte/UpdateTrouble--openfl--8-4-1/tree/master/src](url)

_**My constellation:**_

Windows 10,
Laptop, Acer Aspire ES1-572
Processor: Intel Core i3-6006U CPU @ 2.0 GHz, @ 2.0 GHZ
64-bit, x64 based processor

I can't even tell, if there is something, that I might have done wrong. But if so, the previous versions have been more tolerant against that.

Thank you for taking care.

Kind regards
Arnim

**Comment by EricBishton** [comment]
@hereticByte - Arnim, which function do you have to comment out to stop the crash (or, if it's currently commented out, which do you need to create it).  There's a lot of code in your project that is irrelevant for this bug.  That's just fine; I just need to figure out how to reproduce.

If I am to rebuild this project, I'm going to need the project.xml (and probably the assets that are referenced).  If you don't want to post them publicly, then you can contact me using the contact form at https://bishtonsoftware.com/contact or PM me on Gitter.

**Comment by hereticByte** [comment]
Yes, you're right. I have overseen to upload the _project.xml_ and the _asset_ folder
Now I have completed that.

For me it's time to go to bed now (it's 10:40 pm here).
I am ready for action again tomorrow.

I wish you a good hunt...
;-)

**Comment by imcasper** [comment]
I have crash, then use "lime 7.0.0" or newest and "hxcpp-debugger (protocol 1.1 from git)". I put "trace" around potential stackoverflow error (haxe.Serializer:serialize(v:Dynamic) ):

https://pastebin.com/sXbtfdaf
